streaming: Fail stream plan on stream_mutation_fragments handler in case of error

asias opened the pull request:
The following is observed in pytest:

1) node1, stream master, tried to pull data from node3

2) node3, stream follower, found node1 restarted

3) node3 killed the rpc stream

4) node1 did not get the stream session failure message from node3. This failure message was supposed to kill the stream plan on node1. That's the reason node1 failed the stream session much later at "2024-08-19 21:07:45,539". Note, node3 failed the stream on its side, so it should have sent the stream session failure message.

```
$ cat node1.log |grep f890bea0-5e68-11ef-99ae-e5bca04385fc
INFO  2024-08-19 20:24:01,162 [shard 0:strm] stream_session - [Stream #f890bea0-5e68-11ef-99ae-e5bca04385fc] Executing streaming plan for Tablet migration-ks-index-0 with peers={127.0.34.3}, master
ERROR 2024-08-19 20:24:01,190 [shard 1:strm] stream_session - [Stream #f890bea0-5e68-11ef-99ae-e5bca04385fc] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks=ks, cf=cf, peer=127.0.34.3: seastar::nested_exception: seastar::rpc::stream_closed (rpc stream was closed by peer) (while cleaning up after seastar::rpc::stream_closed (rpc stream was closed by peer))
WARN  2024-08-19 21:07:45,539 [shard 0:main] stream_session - [Stream #f890bea0-5e68-11ef-99ae-e5bca04385fc] Streaming plan for Tablet migration-ks-index-0 failed, peers={127.0.34.3}, tx=0 KiB, 0.00 KiB/s, rx=484 KiB, 0.18 KiB/s

$ cat node3.log |grep f890bea0-5e68-11ef-99ae-e5bca04385fc
INFO  2024-08-19 20:24:01,163 [shard 0:strm] stream_session - [Stream #f890bea0-5e68-11ef-99ae-e5bca04385fc] Executing streaming plan for Tablet migration-ks-index-0 with peers=127.0.34.1, slave
INFO  2024-08-19 20:24:01,164 [shard 1:strm] stream_session - [Stream #f890bea0-5e68-11ef-99ae-e5bca04385fc] Start sending ks=ks, cf=cf, estimated_partitions=2560, with new rpc streaming
WARN  2024-08-19 20:24:01,187 [shard 0: gms] stream_session - [Stream #f890bea0-5e68-11ef-99ae-e5bca04385fc] Streaming plan for Tablet migration-ks-index-0 failed, peers={127.0.34.1}, tx=633 KiB, 26506.81 KiB/s, rx=0 KiB, 0.00 KiB/s
WARN  2024-08-19 20:24:01,188 [shard 0:strm] stream_session - [Stream #f890bea0-5e68-11ef-99ae-e5bca04385fc] stream_transfer_task: Fail to send to 127.0.34.1:0: seastar::rpc::stream_closed (rpc stream was closed by peer)
WARN  2024-08-19 20:24:01,189 [shard 0:strm] stream_session - [Stream #f890bea0-5e68-11ef-99ae-e5bca04385fc] Failed to send: seastar::rpc::stream_closed (rpc stream was closed by peer)
WARN  2024-08-19 20:24:01,189 [shard 0:strm] stream_session - [Stream #f890bea0-5e68-11ef-99ae-e5bca04385fc] Streaming error occurred, peer=127.0.34.1
```

To be safe in case the stream fail message is not received, node1 could fail the stream plan as soon as the rpc stream is aborted in the stream_mutation_fragments handler.

Fixes #20227

Backport to 6.2 6.1.

scylladb-promoter commented:
#### :red_circle: CI State: FAILURE

:white_check_mark: - [Build](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/13915/)
:x: - [Unit Tests Custom](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/13915/)
**The following new/updated tests ran 100 times for each mode:**
:small_blue_diamond: topology_experimental_raft/test_tablets
#### Failed Tests (1/3960):
- [test_decommission_rack_basic.dev.9](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/13915/testReport/junit/topology_experimental_raft/test_tablets/test_decommission_rack_basic_dev_9) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_decommission_rack_basic.dev.9)
#### Build Details:
- Duration: 2 hr 45 min
- Builder: spider1.cloudius-systems.com

asias commented:
> #### 🔴 CI State: FAILURE
> ✅ - [Build](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/13915/) ❌ - [Unit Tests Custom](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/13915/) **The following new/updated tests ran 100 times for each mode:** 🔹 topology_experimental_raft/test_tablets
> 
> #### Failed Tests (1/3960):
> * [test_decommission_rack_basic.dev.9](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/13915/testReport/junit/topology_experimental_raft/test_tablets/test_decommission_rack_basic_dev_9) [🔍](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_decommission_rack_basic.dev.9)

Existing CI issue: https://github.com/scylladb/scylladb/pull/21702
```
          await cql.run_async(f"CREATE KEYSPACE {ks} WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': {rf}}} AND tablets = {{'initial': {initial_tablets}}}")
E           cassandra.AlreadyExists: Keyspace 'test_1734489579114_jlnpi' already exists
```

> #### Build Details:
> * Duration: 2 hr 45 min
> * Builder: spider1.cloudius-systems.com

scylladb-promoter commented:
#### :red_circle: CI State: FAILURE

:white_check_mark: - [Build](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14100/)
:x: - [Unit Tests Custom](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14100/)
**The following new/updated tests ran 100 times for each mode:**
:small_blue_diamond: topology_custom/test_tablets2
#### Failed Tests (1/7920):
- [test_decommission_rack_after_adding_new_rack.dev.3](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/14100/testReport/junit/topology_custom/test_tablets2/test_decommission_rack_after_adding_new_rack_dev_3) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_decommission_rack_after_adding_new_rack.dev.3)
#### Build Details:
- Duration: 5 hr 6 min
- Builder: spider4.cloudius-systems.com

bhalevy commented:
> #### 🔴 CI State: FAILURE
> ✅ - [Build](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14100/) ❌ - [Unit Tests Custom](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14100/) **The following new/updated tests ran 100 times for each mode:** 🔹 topology_custom/test_tablets2
> 
> #### Failed Tests (1/7920):
> * [test_decommission_rack_after_adding_new_rack.dev.3](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/14100/testReport/junit/topology_custom/test_tablets2/test_decommission_rack_after_adding_new_rack_dev_3) [🔍](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_decommission_rack_after_adding_new_rack.dev.3)
> 
> #### Build Details:
> * Duration: 5 hr 6 min
> * Builder: spider4.cloudius-systems.com

@asias please investigate.

@tgrabiec please review the PR

avikivity commented:
What was the state before? Did the sender think the plan completed successfully, or did it just time out later?

asias commented:
> What was the state before? Did the sender think the plan completed successfully, or did it just time out later?

The stream plan will not be reported as successful. It will wait for the other side forever. 

However, I want to mention the PR is just an extra protection. 

Existing code should handle it ok, because

- If the receiver is gone. The failure detector on the sender will figure it out and kill the stream plan.
- If the receiver is not dead, it should send the failure message to fail the stream plan.

The chance of the receiver is not dead and yet it does not send the failure message is very very low.

asias commented:
> > #### 🔴 CI State: FAILURE
> > ✅ - [Build](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14100/) ❌ - [Unit Tests Custom](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14100/) **The following new/updated tests ran 100 times for each mode:** 🔹 topology_custom/test_tablets2
> > #### Failed Tests (1/7920):
> > 
> > * [test_decommission_rack_after_adding_new_rack.dev.3](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/14100/testReport/junit/topology_custom/test_tablets2/test_decommission_rack_after_adding_new_rack_dev_3) [🔍](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_decommission_rack_after_adding_new_rack.dev.3)
> > 
> > #### Build Details:
> > 
> > * Duration: 5 hr 6 min
> > * Builder: spider4.cloudius-systems.com
> 
> @asias please investigate.
> 
> @tgrabiec please review the PR

It is the same previous analysis:
https://github.com/scylladb/scylladb/pull/21960#issuecomment-2550564778

denesb commented:
@asias please rebase, thee are conflicts.

asias commented:
> @asias please rebase, thee are conflicts.

Rebased and pushed.

scylladb-promoter commented:
#### :red_circle: CI State: FAILURE

:white_check_mark: - [Build](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14579/)
:x: - [Unit Tests Custom](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14579/)
**The following new/updated tests ran 100 times for each mode:**
:small_blue_diamond: topology_custom/test_tablets2
#### Failed Tests (2/1981):
- [test_decommission_not_enough_racks.dev.10](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/14579/testReport/junit/topology_custom/test_tablets2/test_decommission_not_enough_racks_dev_10) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_decommission_not_enough_racks.dev.10)
- [test_decommission_not_enough_racks.dev.10](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/14579/testReport/junit/topology_custom/test_tablets2/test_decommission_not_enough_racks_dev_10_2) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_decommission_not_enough_racks.dev.10)
#### Build Details:
- Duration: 6 hr 24 min
- Builder: i-08f21249934abd088 (m5ad.8xlarge)

asias commented:
> #### 🔴 CI State: FAILURE
> ✅ - [Build](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14579/) ❌ - [Unit Tests Custom](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14579/) **The following new/updated tests ran 100 times for each mode:** 🔹 topology_custom/test_tablets2
> 
> #### Failed Tests (2/1981):
> * [test_decommission_not_enough_racks.dev.10](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/14579/testReport/junit/topology_custom/test_tablets2/test_decommission_not_enough_racks_dev_10) [🔍](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_decommission_not_enough_racks.dev.10)
> * [test_decommission_not_enough_racks.dev.10](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/14579/testReport/junit/topology_custom/test_tablets2/test_decommission_not_enough_racks_dev_10_2) [🔍](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_decommission_not_enough_racks.dev.10)
> 
> #### Build Details:
> * Duration: 6 hr 24 min
> * Builder: i-08f21249934abd088 (m5ad.8xlarge)

Known CI issue:

cassandra.AlreadyExists: Keyspace 'test_1737373864369_isvkr' already exists

bhalevy commented:
> > #### 🔴 CI State: FAILURE
> > ✅ - [Build](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14579/) ❌ - [Unit Tests Custom](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14579/) **The following new/updated tests ran 100 times for each mode:** 🔹 topology_custom/test_tablets2
> > #### Failed Tests (2/1981):
> > 
> > * [test_decommission_not_enough_racks.dev.10](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/14579/testReport/junit/topology_custom/test_tablets2/test_decommission_not_enough_racks_dev_10) [🔍](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_decommission_not_enough_racks.dev.10)
> > * [test_decommission_not_enough_racks.dev.10](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/14579/testReport/junit/topology_custom/test_tablets2/test_decommission_not_enough_racks_dev_10_2) [🔍](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_decommission_not_enough_racks.dev.10)
> > 
> > #### Build Details:
> > 
> > * Duration: 6 hr 24 min
> > * Builder: i-08f21249934abd088 (m5ad.8xlarge)
> 
> Known CI issue:
> 
> cassandra.AlreadyExists: Keyspace 'test_1737373864369_isvkr' already exists

That's similar to https://github.com/scylladb/scylladb/issues/22342
to be fixed by https://github.com/scylladb/scylladb/pull/22399

bhalevy commented:
ping @tgrabiec please review

tgrabiec commented:
Rekicked CI

scylladb-promoter commented:
#### :red_circle: CI State: FAILURE

:white_check_mark: - [Build](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/15078/)
:x: - [Unit Tests Custom](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/15078/)
**The following new/updated tests ran 100 times for each mode:**
:small_blue_diamond: topology_custom/test_tablets2
#### Failed Tests (40/990):
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.1](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15078/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_1) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.1)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.10](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15078/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_10) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.10)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.2](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15078/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_2) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.2)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.3](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15078/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_3) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.3)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.4](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15078/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_4) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.4)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.5](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15078/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_5) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.5)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.6](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15078/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_6) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.6)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.7](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15078/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_7) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.7)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.8](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15078/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_8) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.8)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.9](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15078/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_9) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.9)
#### Build Details:
- Duration: 4 hr 0 min
- Builder: i-0aabb0a9b770cd346 (m5ad.8xlarge)

tgrabiec commented on test/topology_custom/test_tablets2.py:
```diff
@@ -243,6 +243,100 @@ async def check():
+
+    await manager.api.enable_injection(servers[0].ip_addr, "stream_session_ignore_failed_message", one_shot=True)
+    await manager.api.enable_injection(servers[1].ip_addr, "stream_session_ignore_failed_message", one_shot=True)
+    await manager.api.enable_injection(servers[1].ip_addr, "stream_mutation_fragments_rx_error", one_shot=True)
```
You're assuming that server[1] will be the receiving server (replica[1]), which is not guaranteed.

asias replied:
Correct. I updated the test to make sure server[0] is the source.

asias commented:
Changes in cdb7841a38c1b79f14dbe35d4f2146b7ae868ba7:
- Rebased to master
- Modified test to make sure servers[0] is the src node to move the tablet from

scylladb-promoter commented:
#### :red_circle: CI State: FAILURE

:white_check_mark: - [Build](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/15115/)
:x: - [Unit Tests Custom](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/15115/)
**The following new/updated tests ran 100 times for each mode:**
:small_blue_diamond: topology_custom/test_tablets2
#### Failed Tests (40/990):
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.1](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15115/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_1) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.1)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.10](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15115/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_10) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.10)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.2](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15115/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_2) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.2)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.3](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15115/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_3) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.3)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.4](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15115/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_4) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.4)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.5](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15115/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_5) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.5)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.6](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15115/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_6) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.6)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.7](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15115/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_7) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.7)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.8](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15115/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_8) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.8)
- [test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.9](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/15115/testReport/junit/topology_custom/test_tablets2/test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang_debug_9) [:mag:](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_streaming_rx_error_no_failed_message_no_fail_stream_plan_hang.debug.9)
#### Build Details:
- Duration: 1 hr 48 min
- Builder: spider4.cloudius-systems.com

asias commented:
Changes in 56238b83e483be506bb5fd5a3e9ef304f7a825d1
- Disable file stream since we are testing classic streaming in the test and file stream is on by default with recent master
